TileFormats/Instanced3DModel/README.md

```diff
@@ -22,65 +22,180 @@ A tile is composed of a header section immediately followed by a body section.
+| `NORMAL_RIGHT_OCT32P` | `uint16[2]` | An oct-encoded unit vector with 32-bits of precision defining the `right` direction for the orientation of the instance. Must be orthogonal to `up`. | :red_circle: No <br> :large_blue_diamond: Depends on `NORMAL_RIGHT_OCT32P` |
+| `SCALE` | `float32` | A number defining a scale to apply to all axes of the instance. | :red_circle: No |
+| `SCALE_NON_UNIFORM` | `float32[3]` | A 3-component array of numbers defining the scale to apply to the `x`, `y`, and `z` axes of the instance. | :red_circle: No |
+| `BATCH_ID` | `unit16` | The `batchId` of the instance that can be used to retrieve metadata from the `Batch Table`. | :red_circle: No |
+
+#### Global Semantics
+
+These semantics define global properties for all instances.
+
+| Semantic | Data Type | Description | Required |
+| --- | --- | --- | --- |
+| `INSTANCES_LENGTH`| `uint32` | The number of instances to generate. The length of each array value for an instance semantic should be equal to this. | :white_check_mark: Yes |
+| `QUANTIZED_VOLUME_OFFSET` | `float32[3]` | A 3-component array of numbers defining the offset for the quantized volume. | :red_circle: No |
+| `QUANTIZED_VOLUME_SCALE` | `float32[3]` | A 3-component array of numbers defining the scale for the quantized volume. |:red_circle: No |
+
+### Instance Orientation
+
+An instance's orientation is defined by an orthonormal basis created by an `up` and `right` vector. If `NORMAL_UP` and `NORMAL_RIGHT` or `NORMAL_UP_OCT32P` and `NORMAL_RIGHT_OCT32P` are not present,
+the instance will default to the `east/north/up` reference frame's orientation for the instance's Cartographic position (`x`, `y`, `z` converted to `longitude` and `latitude` on the `WGS84` ellipsoid).
+
+The `x` vector in the standard basis maps onto the `right` vector in the transformed basis, and the `y` vector maps on to the `up` vector.
+The `z` vector would map onto a `forward` vector, but it is omitted because it will always be the cross product of `right` and `up`.
+
+**Figure 2**: A box in the standard basis.
+
+![box standard basis](figures/box-standard-basis.png)
+
+**Figure 3**: A box transformed into a rotated basis.
+
+![box rotated basis](figures/box-rotated-basis.png)
+
+#### Oct-encoded Normal Vectors
+
+If `NORMAL_UP` and `NORMAL_RIGHT` are not defined for an instance, its orientation may be stored as oct-encoded normals in `NORMAL_UP_OCT32P` and `NORMAL_RIGHT_OCT32P`.
+
+These define `up` and `right` using the oct-encoding described in
+[*A Survey of Efficient Representations of Independent Unit Vectors* by Cigolle et al.](http://jcgt.org/published/0003/02/01/).
+
+An implementation for encoding and decoding these unit vectors can be found in Cesium's
+[AttributeCompression](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/AttributeCompression.js)
+module.
+
+### Instance Position
+
+`POSITION` defines the location where an instance should be placed in Cartesian space.
```

Comment:
What space?  Be precise here with respect to #98.  @lilleyse please help provide language if needed.